packages/react-centra-checkout/src/server/createGetCentraWebhookEvents.ts

```diff
@@ -0,0 +1,166 @@
+import type { NextApiRequest } from 'next'
+import crypto from 'node:crypto'
+import { z } from 'zod'
+
+const sign = (secret: string, dataString: string) => {
+  const hmac = crypto.createHmac('sha256', secret)
+
+  hmac.update(dataString)
+
+  return hmac.digest('hex')
+}
+
+/**
+ * Parse a named parameters string into an object.
+ * @example
+ * parseNamedParametersString('foo=bar,baz=qux') // \{ foo: 'bar', baz: 'qux' \}
+ */
+const parseNamedParametersString = (input: string): Record<string, string> => {
```

Comment:
Keep in mind for the foreseeable future; `objectFromURLSearchParams` PR may solve this similarly. Consider seeing if it can be applicable here for code-reuse.

Reply:
Noted!
I tried if i could use `objectFromUrlSearchParams` and see if it "just works", which, unfortunately, it doesn't.
Would be nice to get a shared solution, though, since the only thing that differs interface-wise is a separator.